File: medplum-java/medplum-model/src/main/java/com/fhir/Location_Position.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableLocation_Position.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface Location_Position {
  Optional<decimal> altitude();

  Optional<List<Extension>> extension();

  Optional<List<Extension>> modifierExtension();

  Optional<decimal> longitude();

  Optional<decimal> latitude();

  Optional<String> id();

  static ImmutableLocation_Position.Builder builder() {
    return ImmutableLocation_Position.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/Location_Position.ts
import { decimal } from "./decimal";
import { Extension } from "./Extension";

export interface Location_Position {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    longitude: decimal | null | undefined;
    latitude: decimal | null | undefined;
    altitude: decimal | null | undefined;
}
